client/client.go

```diff
@@ -109,9 +109,9 @@ func (c *LogClient) ListByIndex(ctx context.Context, start, count int64) ([]*tri
 	return resp.Leaves, nil
 }
 
-// waitForRootUpdate repeatedly fetches the Root until the TreeSize changes
+// WaitForRootUpdate repeatedly fetches the Root until the TreeSize changes
 // or until ctx times out.
-func (c *LogClient) waitForRootUpdate(ctx context.Context) error {
+func (c *LogClient) WaitForRootUpdate(ctx context.Context) error {
```

Comment:
Would it be handy for some callers to have the new root returned?

Reply:
Indeed it would!